tests/yast2_gui/yast2_firewall_set_default_zone_prepare.pm

```diff
@@ -0,0 +1,35 @@
+# SUSE's openQA tests
+#
+# Copyright 2022 SUSE LLC
+# SPDX-License-Identifier: FSFAP
+
+# Summary: This module should schedule before yast2_firewall_set_default_zone.pm on SLES productor since
+# TW set default zone with iface together but sle not, so we need update iface's zone to trusted zone firstly
+# Maintainer: QE YaST <qa-sle-yast@suse.de>
+
+use base "y2_module_guitest";
+use strict;
+use warnings;
+use testapi;
+use utils;
+use network_utils 'iface';
+use YaST::Module;
+
+sub run {
+    my $self = shift;
+    my $iface = iface();
+    my %setting = (device => $iface, zone => 'trusted');
```

Comment:
not sure why to use a hash, you already need a variable above for $iface, so just adding another one with $zone, but you can also hardcode in the calls and only have $iface, not zone.